<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusDataMonitorForm</class>
 <widget class="QWidget" name="BusDataMonitorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_txshow">
       <property name="text">
        <string>发送数据监控</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_rxshow">
       <property name="text">
        <string>采集数据监控</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QMdiArea" name="mdiArea"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
